src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -98,8 +88,98 @@ public function get_arguments() {
 		$arguments['name']                          = esc_html__( 'Events List V2', 'the-events-calendar' );
 		$arguments['widget_options']['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
 
+		// Setup default title.
+		$arguments['title'] = __( 'Upcoming Events', 'the-events-calendar' );
+
+		// Setup admin fields.
+		$arguments['admin_fields'] = $this->get_admin_fields();
+
+		// Add the Widget to the arguments to pass to the admin template.
+		$arguments['widget_obj'] = $this;
+
 		$arguments = wp_parse_args( $arguments, $this->get_default_arguments() );
 
 		return $this->filter_arguments( $arguments );
 	}
+
+	/**
+	 * The function for saving widget updates in the admin section.
+	 *
+	 * @param array $new_instance
+	 * @param array $old_instance
+	 *
+	 * @return array The new widget settings.
+	 */
+	public function update( $new_instance, $old_instance ) {
+		$instance = $old_instance;
+
+		/* Strip tags (if needed) and update the widget settings. */
+		$instance['title']                = strip_tags( $new_instance['title'] );
```

Comment:
:warning: `strip_tags()` does not strip CSS and JS in between the script and style tags. Use `wp_strip_all_tags()` to strip all tags.

<sub>hash: e5344a27af85e0292d1d1483b8eca12a</sub>